packages/components/src/FormatFile/FormatFile.mdx

```diff
@@ -35,9 +35,10 @@ import { FormatFile } from "@jobber/components/FormatFile";
         name: "myballisbigandroundIamrollingitontheground.png",
         type: "image/png",
         size: 213402324,
-        progress: 0.3,
+        progress: 0.4,
         src: () => Promise.resolve("https://source.unsplash.com/250x250"),
       }}
+      display="thumbnail"
       onDelete={() => alert("🧨BALEETED!💥")}
```

Comment:
I know this is pre-existing but can we change all of these examples to say "Deleted" instead of "🧨BALEETED!💥" ?